plugins/mcreator-core/procedures/entity-freeze.json

```diff
@@ -0,0 +1,30 @@
+{
+   "message0": "Freeze %2 for %1 ticks",

```

Comment:
Put messages inside text files, so they can be translated.

Reply:
Sure thing, gonna work on that!

Reply:
Could you give me an example on where to implement that?

Reply:
https://github.com/MCreator/MCreator/blob/master/plugins/mcreator-localization/lang/texts.properties#L762